monitoring/dashboard.json

```diff
@@ -0,0 +1,627 @@
+                "w": 24,
+                "x": 0,
+                "y": 9
+            },
+            "id": 10,
+            "panels": [],
+            "title": "Latency",
+            "type": "row"
+        },
+        {
+            "datasource": null,
+            "fieldConfig": {
+                "defaults": {
+                    "color": {
+                        "mode": "palette-classic"
+                    },
+                    "custom": {
+                        "axisLabel": "",
+                        "axisPlacement": "auto",
+                        "barAlignment": 0,
+                        "drawStyle": "line",
+                        "fillOpacity": 0,
+                        "gradientMode": "none",
+                        "hideFrom": {
+                            "legend": false,
+                            "tooltip": false,
+                            "viz": false
+                        },
+                        "lineInterpolation": "linear",
+                        "lineWidth": 1,
+                        "pointSize": 5,
+                        "scaleDistribution": {
+                            "type": "linear"
+                        },
+                        "showPoints": "auto",
+                        "spanNulls": false,
+                        "stacking": {
+                            "group": "A",
+                            "mode": "none"
+                        },
+                        "thresholdsStyle": {
+                            "mode": "off"
+                        }
+                    },
+                    "mappings": [],
+                    "thresholds": {
+                        "mode": "absolute",
+                        "steps": [
+                            {
+                                "color": "green",
+                                "value": null
+                            },
+                            {
+                                "color": "red",
+                                "value": 80
+                            }
+                        ]
+                    },
+                    "unit": "ms"
+                },
+                "overrides": []
+            },
+            "gridPos": {
+                "h": 8,
+                "w": 12,
+                "x": 0,
+                "y": 10
+            },
+            "id": 12,
+            "options": {
+                "legend": {
+                    "calcs": [],
+                    "displayMode": "list",
+                    "placement": "bottom"
+                },
+                "tooltip": {
+                    "mode": "single"
+                }
+            },
+            "targets": [
+                {
+                    "exemplar": true,
+                    "expr": "(sum(rate(http_request_duration_seconds_sum{namespace=\"${namespace}\", job=\"${job}\"}[10m]) * 1000)) / (sum(rate(http_request_duration_seconds_count{namespace=\"${namespace}\", job=\"${job}\"}[10m]) * 1000))",
```

Comment:
same thing about 10m.
also, just thinking: should we use the same averaging window for dashboard and alerts?

Note that we may use a variable to avoid repeating this value, if it makes sense....

Reply:
no for alerts we want to have the _spikes_ when some latency reach them and we want them for a short period of time, here people may look at some graphs over a week period, or day period and so we want at very large value here (from minutes to hours).